src/components/Questions/AnswerInput.vue

```diff
@@ -82,19 +113,41 @@ export default {
 
 	data() {
 		return {
+			localAnswer: this.answer,
 			queue: new PQueue({ concurrency: 1 }),
-
-			// As data instead of Method, to have a separate debounce per AnswerInput
-			debounceUpdateAnswer: pDebounce(function(answer) {
-				return this.queue.add(() => this.updateAnswer(answer))
-			}, 500),
 		}
 	},
 
 	computed: {
+		ariaLabel() {
+			if (this.local) {
+				return t('forms', 'Add a new answer')
+			}
+			return t('forms', 'An answer for the {index} option', { index: this.index + 1 })
```

Comment:
```suggestion
			return t('forms', 'An answer for the option {index}', { index: this.index + 1 })
```
Wouldn't this result in "An answer for 1 option" which should be "1st option"?

Reply:
Basically copied / moved from here:
https://github.com/nextcloud/forms/pull/2092/files#diff-644e72f1327ea20409e3f75541d13240c8d29e59ddfdfd14e3c261d3df3ca7d1L7

But yes looks weird